repo/packages/A/arangodb/0/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "arangodb",
+  "version": "0.2.0",
+  "scm": "https://github.com/arangodb/arangodb-mesos.git",
+  "maintainer": "info@arangodb.com",
+  "description": "A distributed free and open-source database with a flexible data model for documents, graphs, and key-values. Build high performance applications using a convenient SQL-like query language or JavaScript extensions.",
+  "framework": true,
+  "tags": [ "arangodb", "NoSQL", "database", "framework" ],
+  "images": {
+    "icon-small": "https://raw.githubusercontent.com/arangodb/arangodb-dcos/master/icons/arangodb_small.png",
+    "icon-medium": "https://raw.githubusercontent.com/arangodb/arangodb-dcos/master/icons/arangodb_medium.png",
+    "icon-large": "https://raw.githubusercontent.com/arangodb/arangodb-dcos/master/icons/arangodb_large.png"
+  },
+  "preInstallNotes": "Note that the ArangoDB DCOS Service is beta and there may be bugs, incomplete features, incorrect documentation or other discrepancies.",
```

Comment:
the focus of the preinstall notes should be to inform the users what resources are necessary for this application to run.